test/wdio/lifecycle-basic/cmp-a.test.tsx

```diff
@@ -11,37 +11,32 @@ describe('lifecycle-basic', function () {
   it('fire load methods in order', async () => {
     await $('lifecycle-basic-a').waitForExist();
 
-    await browser.pause(200);
-
-    let loads = await $('.lifecycle-loads-a').$$('li');
-    await expect(loads.length).toBe(6);
-
+    let loads = $('.lifecycle-loads-a').$$('li');
+    await expect(loads).toBeElementsArrayOfSize(6);
     await expect(loads[0]).toHaveText('componentWillLoad-a');
     await expect(loads[1]).toHaveText('componentWillLoad-b');
     await expect(loads[2]).toHaveText('componentWillLoad-c');
     await expect(loads[3]).toHaveText('componentDidLoad-c');
     await expect(loads[4]).toHaveText('componentDidLoad-b');
     await expect(loads[5]).toHaveText('componentDidLoad-a');
 
-    let updates = await $('.lifecycle-updates-a').$$('li');
-    await expect(updates).not.toBeExisting();
+    let updates = $('.lifecycle-updates-a').$$('li');
+    await expect(loads).toBeElementsArrayOfSize(6);
```

Comment:
Why do we change the assertion here to be against the field in `loads` instead of `updates` here?

Reply:
Oh, my bad! It is actually suppose to be `await expect(updates).toBeElementsArrayOfSize(0);`